format/mpeg/testdata/mp4-he-aac-hierarchical-signalling.fqtest

```diff
@@ -0,0 +1,1038 @@
+$ fq -d mp4 dv mp4-he-aac-hierarchical-signalling
```

Comment:
if there are specific commands or steps to re-produce the test file i sometimes put them in a comment at the top